examples/data-sources/source_secret/data_source_cluster.tf

```diff
@@ -0,0 +1,12 @@
+# Read Tanzu Mission Control source secret : fetch cluster source secret details
```

Comment:
Add resource templates for all examples as well

Reply:
for data source only these two (cluster, cluster_group) examples templates is required.

Reply:
I meant resource templates corresponding to each example file